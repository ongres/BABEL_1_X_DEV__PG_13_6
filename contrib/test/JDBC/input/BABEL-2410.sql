USE master
go

-- Should be blocked
CREATE ROLE babel_2410_role
go

DROP ROLE babel_2410_role
go
