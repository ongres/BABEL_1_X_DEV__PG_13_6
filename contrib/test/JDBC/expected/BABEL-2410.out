USE master
go

-- Should be blocked
CREATE ROLE babel_2410_role
go
~~ERROR (Code: 33557097)~~

~~ERROR (Message: 'CREATE ROLE' is not currently supported in Babelfish)~~


DROP ROLE babel_2410_role
go
~~ERROR (Code: 33557097)~~

~~ERROR (Message: 'DROP ROLE' is not currently supported in Babelfish)~~

